src/VisualStudio/Core/Def/Implementation/VirtualMemoryNotificationListener.cs

```diff
@@ -21,6 +22,9 @@ namespace Microsoft.VisualStudio.LanguageServices
     [Export, Shared]
     internal sealed class VirtualMemoryNotificationListener : ForegroundThreadAffinitizedObject, IVsBroadcastMessageEvents
     {
+        // memory threshold to turn off full solution analysis - 200MB
+        private const long MemoryThreshold = 200 * 1024 * 1024;
```

Comment:
What's the intention here?  Do you think this will be more or less aggressive that it is today (and why)?